cmd/server/config.go

```diff
@@ -23,8 +22,8 @@ var (
 
 type config struct {
 	Mode             string `json:"mode"`
-	RPCEndpoint      string `json:"rpc_endpoint"`
-	IndexerEndpoint  string `json:"indexer_endpoint"`
+	RPCBaseUrl       string `json:"rpc_base_url"`
+	IndexerBaseUrl   string `json:"indexer_base_url"`
```

Comment:
It seems to me that specifying these are really base urls avoids confusion with avalanchego indexer endpoint path and highlight the flexibility of using two different sources for data